api/src/main/java/edu/cornell/mannlib/vitro/webapp/dynapi/components/Action.java

```diff
@@ -59,7 +66,14 @@ public String getKey() {
 
     @Override
     public boolean isValid() {
-        return true;
+        boolean result = false;
+        try {
+            computeParams();
```

Comment:
If this computeParams() should always be invoked before validation over validators assigned to requestedParams, then this code will be repeated in all subclasses of Operations. Maybe, we can move part of that to Operation class and define there method executeValidators which should be only override.

Reply:
No, it is not directly related to validation. I think that action should be validated only once on action load